### PR TITLE
fixed labelType none still shows label when hovering over node

### PR DESCRIPTION
**Pull request by dtsuper3**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/symbols/Node.tsx [hunk 1]
```diff
@@ -306,6 +306,7 @@ export const Node: FC<NodeProps> = ({
 
   const labelComponent = useMemo(
     () =>
+      labelVisible &&
```
Comment (amcdnl) [inline src/symbols/Node.tsx:309]: @ghsteff - can u confirm this change won't regress any of the other features we added?

**Review by ghsteff — APPROVED** [review]
Looks good!

**Comment by Obaydahm** [comment]
@ghsteff @amcdnl is it possible to possible a new version that includes these changes? I can see that the latest version is 2 months old. Thanks in advance!

**Comment by ghsteff** [comment]
@Obaydahm yep! Released in 4.19.3

**Pull request merged**